products/sle/main.pm

```diff
@@ -371,8 +371,15 @@ sub load_boot_tests() {
         loadtest "installation/bootloader_ofw.pm";
     }
     elsif (get_var("UEFI") || is_jeos) {
+        if (check_var("BACKEND", "svirt")) {
+            loadtest "installation/bootloader_svirt.pm";
+        }
         # TODO: rename to bootloader_grub2
-        loadtest "installation/bootloader_uefi.pm";
+        # Unless GRUB2 supports framebuffer on Xen PV (bsc#961638), grub2 tests
+        # has to be skipped there.
+        if (!((get_var('VIRSH_VMM_FAMILY') eq 'xen') && (get_var('VIRSH_VMM_TYPE') eq 'linux'))) {
```

Comment:
this will give you a warning about undefined variables. Plus you're duplicating check_var